schemas/components/schemas/WebhookVelocity.yaml

```diff
@@ -0,0 +1,23 @@
+type: object
```

Comment:
Lets watch out for conflicts with this PR: https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/pull/166

Reply:
Sure, once it's merged I'll update this PR accordingly.